src/utils/iframe.js

```diff
@@ -13,54 +13,50 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import {Deferred} from '../../vendor/ampproject/amphtml/src/utils/promise';
-
-/** @private {?boolean} */
-let srcdocSupported = null;
+import once from 'lodash.once';
 
 /**
  * Returns `true` if srcdoc is supported. https://caniuse.com/#search=srcdoc
  * @return {boolean}
  */
-export function isSrcdocSupported() {
-  if (srcdocSupported === null) {
-    srcdocSupported = 'srcdoc' in HTMLIFrameElement.prototype;
-  }
-  return srcdocSupported;
-}
+const isSrcdocSupported = once(() => 'srcdoc' in HTMLIFrameElement.prototype);
```

Comment:
@calebcordry Do we even really need to `once` this? This effectively gets executed once, and looking up an object property (e.g. `'srcdoc' in...`) is `O(1)`.

Reply:
We don't, removed 😄

Reply:
SGTM, technically i think `in` is O(n) but n is small :)